frontend/src/pages/Overview.js

```diff
@@ -0,0 +1,66 @@
+import SessionBoard from '../components/SessionBoard';
+import PageLayout from '../components/PageLayout';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import ButtonTabBottom from '../components/ButtonTabBottom';
+import { Redirect } from 'react-router-dom';
+
+export default function Overview({ jwtToken }) {
+    const sessions = [
```

Comment:
Maybe extract to an extra file, even if it is only temporary it hinders readability :)

Reply:
it will be gone as soon as I get the real sessions from the backend ^^'